### ISSUE TITLE
yolov5 on Intel compute stick2?

### ISSUE THREAD

**Issue by sujithgunturu** [ultralytics/yolov5#3063]
## ❔Question
How to run my yolov5 with weights "best.pt" on intel compute stick. 
## Additional context
I have trained yolo on 3200 images with yolo large model.  Then converted ".pt" to ".onnx" with "export.py". Later converted ".onnx" format to ".bin" and ".xml"  format with openvino's model optimizer to run the model on intel compute stick. All the predictions done are incorrect and make no sense. for example: confidence and class label are negative. how to do it in right way. 

I have trained my yolo in this way
python train.py --img 1024 --batch 24 --epochs 100 --data wheat.yaml --cfg models/yolov5l.yaml --name wml

**Comment by glenn-jocher** [comment]
@sujithgunturu we don't have an NCS2 tutorial, but one thing I would advise you is to first try to deploy an official model before trying to deploy your custom trained model. This way you can isolate sources of error and have a better understanding of where to focus your debugging efforts.

**Comment by glenn-jocher** [comment]
@sujithgunturu good news 😃! Your original issue may now be fixed ✅ in PR #6057. This PR adds native YOLOv5 OpenVINO export:

```
python export.py --include openvino
```
![image](https://user-images.githubusercontent.com/26833433/147145516-b4f09a44-5679-472a-b512-97c51adffefc.png)


To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by Averen19** [comment]
Hi, I have trained a custom dataset and would like to deploy on the NCS2 using Raspberry Pi. I converted the .pt file to .xml with the python export.py --include openvino code. However, my class_ids name are wrong as it still shows the original class ids like people and bicycle for class ids 0 and 1. Any solution?

**Comment by glenn-jocher** [comment]
@Averen19 answered in https://github.com/ultralytics/yolov5/issues/7076

**Comment by Averen19** [comment]
hey @sujithgunturu, did you manage to run your yolo model on ncs2? If you did, do you mind sharing the code to deploy the model after converting to ir format?
